### ISSUE TITLE
shufflenetv2 pretrained model

### ISSUE THREAD

**Issue by zzchust** [PaddlePaddle/models#2194]
The released pre-trained model and accuracy is for ShuffleNetV2_x0_5 or ShuffleNetV2_x1_0 or ShuffleNetV2_x1_5 or ShuffleNetV2_x2_0 ?

and the pretrained model is configured with "model_name" or "model"  ?

**Comment by shippingwang** [comment]
@zzchust  for the regular one: 1X, and you should update the code, We don't use models_name or model to specify category anymore.
